### ISSUE TITLE
Stop warning about protected (maybe even private) access modifiers

### ISSUE THREAD

**Issue by eamodio** [MeirionHughes/aurelia-template-lint#165]
Since access modifiers are typescript only and provide no restrictions for Aurelia or at runtime, it seems (to me at least) that there shouldn't be warnings about them. Or at least a configuration option to turn them off.

As an example I often use `protected` as a modifier for things I want the view/template from having access to, but if someone were to get access to the VM directly I wouldn't want them to have access to (or at least it pollutes the api with things specific only for the view/template).

**Comment by MeirionHughes** [comment]
```ts
config.aureliaBindingAccessOpts.restrictedAccess = [];
```
 https://github.com/MeirionHughes/aurelia-template-lint/blob/develop/source/config.ts#L142

**Comment by eamodio** [comment]
@MeirionHughes doh -- sorry about that :(

**Comment by MeirionHughes** [comment]
Don't worry about it. :)
